x/staking/types/validator_test.go

```diff
@@ -245,14 +245,9 @@ func TestPossibleOverflow(t *testing.T) {
 		msg, newValidator.DelegatorShareExRate())
 }
 
-func TestHumanReadableString(t *testing.T) {
+func String(t *testing.T) {
 	validator := NewValidator(addr1, pk1, Description{})
-
-	// NOTE: Being that the validator's keypair is random, we cannot test the
-	// actual contents of the string.
-	valStr, err := validator.HumanReadableString()
-	require.Nil(t, err)
-	require.NotEmpty(t, valStr)
+	require.NotEmpty(t, validator.String())
```

Comment:
I'd test whether the return value matches the expected template

Reply:
I hear your point here. The tests are left over from when some of these string functions used to return errors. Do you have an issue with keeping the string functions not as templates?

Reply:
I'm going to go ahead and remove this test as well.

Reply:
Not really. Fine for me either way.